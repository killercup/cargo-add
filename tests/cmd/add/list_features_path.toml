bin.name = "cargo-add"
args = ["add", "cargo-list-test-fixture-dependency", "../dependency"]
status = "success"
fs.sandbox = true
fs.cwd = "list_features_path.in/primary"

[env.add]
CARGO_IS_TEST="1"
