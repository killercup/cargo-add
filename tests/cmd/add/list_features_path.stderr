      Adding cargo-list-test-fixture-dependency v99999.0.0 to dependencies. 
      Adding your-face v0.1.3 to dependencies. 
             Available features: 
             - default
             - nose
             - mouth
             - eyes
